src/main/java/io/vertx/core/json/Json.java

```diff
@@ -51,6 +52,10 @@
     prettyMapper.configure(JsonParser.Feature.ALLOW_COMMENTS, true);
     prettyMapper.configure(SerializationFeature.INDENT_OUTPUT, true);
 
+    List<Module> modules = ObjectMapper.findModules(Thread.currentThread().getContextClassLoader());
```

Comment:
Fixed with using `Json.class.getClassLoader` if TCCL is null. Json's CL can't be null since it's not primitive or void type and isn't loaded by bootstrap classloader.